### PR TITLE
Update gradle files

### PR DESCRIPTION
**Pull request by VicV**
Updated plenty of gradle things.

Changed Maps to HashMaps (we're specific).

Android Studio automatically changed the spacing of the integration file.

### CODE REVIEW

**Comment by f2prateek** [comment]
> Android Studio automatically changed the spacing of the integration file.

You can use our code styles here https://github.com/segmentio/java-code-styles

**Comment by VicV** [comment]
@f2prateek much cleaner.

File: src/test/java/com/segment/analytics/android/integrations/taplytics/TaplyticsTest.java [hunk 1]
```diff
@@ -75,14 +73,12 @@
   @Test public void initialize() {
     ValueMap settings = new ValueMap() //
         .putValue("apiKey", "foo") //
-        .putValue("sessionMinutes", 20)
-        .putValue("liveUpdate_v2", "true") //
-        .putValue("shakeMenu_v2", "false")
-        .putValue("turnMenu_v2", "default");
+        .putValue("sessionMinutes", 20).putValue("liveUpdate_v2", "true") //
```
Comment (f2prateek) [inline src/test/java/com/segment/analytics/android/integrations/taplytics/TaplyticsTest.java]: Can we leave these as separate lines?

Comment (VicV) [reply src/test/java/com/segment/analytics/android/integrations/taplytics/TaplyticsTest.java]: Not according to your code styles. I just ran the auto formatter over it.

Comment (f2prateek) [reply src/test/java/com/segment/analytics/android/integrations/taplytics/TaplyticsTest.java]: Yeah IntelliJ isn't the smartest when it comes to unwrapping these. That's why we've been migrating to https://github.com/f2prateek/gradle-android-javafmt-plugin, but will take us some time to get all the repos.

**Comment by VicV** [comment]
@f2prateek forced the change anyway.

**Pull request closed without merging**